test/e2e/e2e-test/src/test/java/org/apache/skywalking/e2e/storage/StorageE2E.java

```diff
@@ -156,6 +167,54 @@ void serviceInstanceTopo() throws Exception {
         verifyServiceInstanceRelationMetrics(topology.getCalls());
     }
 
+    @RetryableTest
+    void addUITemplate() throws Exception {
+        try {
+            TemplateChangeStatus templateChangeStatus = graphql.addTemplate(
+                new DashboardSetting()
+                    .name("test-ui-config-1")
+                    .active(true)
+                    .configuration("{}")
+                    .type(TemplateType.DASHBOARD)
+            );
+            LOGGER.info("add template = {}", templateChangeStatus);
+        } catch (Exception e) {
+            LOGGER.error("add ui template error.", e);
+        }
+        verifyTemplates("expected/storage/dashboardConfiguration.yml");
+    }
+
+    @RetryableTest
+    void changeTemplate() throws Exception {
```

Comment:
Seems this case depends on `addUITemplate`, it's a bad idea that one test case depends on another, please add an individual template in this case for testing modification (`changeTemplate`), BTW, I'm wondering whether `updateTemplate` in the backend query plugin is better or not, `update` is more widely used in the `CRUD` operations @wu-sheng

Reply:
I don't want to discuss about the language culture thing, as these two words are not that different. You could argue about add, insert or create too. As creating is a part of CRUD. 

The point is, unless we want to set up term rules in order to keep all names consistent in the project, don't know how to say which one is better.